inspec/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM     centos:centos7
```

Comment:
Is this the Dockerfile that we'll run as a cron job on the admin VM's?  Can we register it with quay, and make sure it builds?

Reply:
Already taken care of https://quay.io/repository/cdis/inspec-aws